en/docs/how-tos/endpoints/overriding-endpoint-information/overriding-endpoints-for-imported-apis.md

```diff
@@ -64,18 +86,21 @@ Follow the instructions below to override the production or sandbox endpoint(s).
    
     ``` text tab="Example 1 - single endpoint"
     gateway /home/user/petstore-project/target/petstore-project.jar  
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com"
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin" 
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_basic_password="admin"
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com"
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin" 
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_basic_password="admin"
     ```
     
     ``` text tab="Example 2 - multiple endpoints"
     gateway /home/user/petstore-project/target/petstore-project.jar 
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com" 
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_1="http://support.wso2.com"
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin" 
-    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_basic_password="admin"
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com" 
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_1="http://support.wso2.com"
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin" 
+    --api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_basic_password="admin"
     ```
+    
+    !!! Note
```

Comment:
Isn't this note supposed to go under the topic "Overriding endpoints using system properties"

Reply:
or else, we can link it here or move that "Overriding endpoints using system properties" section's content to here.